python/lsst/ts/ofc/state_estimator.py

```diff
@@ -105,7 +105,7 @@ def dof_state(
 
         # Evaluate sensitivity matrix at sensor positions
         sensitivity_matrix = self.dz_sensitivity_matrix.evaluate(
-            field_angles, rotation_angle
+            field_angles, -rotation_angle
```

Comment:
Please add the comment to explain this sign change. Thanks!

Reply:
With the change of the coordinate systems in the XY plane, we need to ensure that the rotation of the Zouble Zernike object in the UV plane is opposite to that one.

Reply:
Please put the above comment in the code. Thanks!